tests/test_supervision.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pytest
+import unittest
+from datetime import timedelta, datetime
+from mock import Mock
+from protoactor.supervision import OneOfOneStrategy, SupervisorDirective, Supervisor
+from protoactor.pid import PID
+from protoactor.process_registry import ProcessRegistry
+from protoactor.restart_statistics import RestartStatistics
+from protoactor.process import LocalProcess
+from protoactor.mailbox import MailBox
+from protoactor.messages import ResumeMailbox, Restart, Stop
+
+@pytest.fixture(scope='module', )
+def supervisor_data():
+    supervisor = Supervisor()
+    mailbox = MailBox()
+    local_process = LocalProcess(self.mailbox)

```

Comment:
You need to remove `self` from `self.mailbox` and `self.local_process`. it fails due to that.